src/qml/qmllibraryproxy.h

```diff
@@ -21,7 +21,7 @@ class QmlLibraryProxy : public QObject {
 
   private:
     std::shared_ptr<Library> m_pLibrary;
-    parented_ptr<QmlLibraryTrackListModel> m_pModel;
+    QmlLibraryTrackListModel* m_pModel;
```

Comment:
Add a comment to prevent a rollback by someone else in the future? Maybe also rename it as `m_pModelProperty`?

Reply:
done.